package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rdfjs/types",
-  "version": "2.0.0",
+  "version": "0.0.0",
```

Comment:
I pushed back to `0.0` since this package was never released.

As you see in `.changesets/lemon-comics-knock.md`, this PR will cause a major bump, thus the next package would be `1.0`